src/SecTester.Reporter/DefaultFormatter.cs

```diff
@@ -0,0 +1,98 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using SecTester.Scan.Models;
+
+namespace SecTester.Reporter;
+
+internal class DefaultFormatter : Formatter
+{
+  public const char NewLine = '\n';
+  private const char BulletPoint = '●';
+  private const char Tabulation = '\t';
+  private const string TemplateBody = @"Issue in Bright UI:   {0}
+Name:                 {1}
+Severity:             {2}
+Remediation:
+{3}
+Details:
+{4}";
+  private const string TemplateExtraDetails = "\nExtra Details:\n{5}";
+  private const string TemplateReferences = "\nReferences:\n{6}";
+
+  private static readonly IEnumerable<Comment> EmptyComments = new List<Comment>();
+  private static readonly IEnumerable<string> EmptyResources = new List<string>();
```

Comment:
So why does it need? It seems you forgot what we discussed recently about redundant optimization in place of readability and cleanness (e.g. `issue.Comments ?? Array.Empty<Comment>()`)